frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "@nextui-org/react": "^2.3.6",
     "@radix-ui/react-dialog": "^1.1.2",
     "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-popover": "^1.1.2",
```

Comment:
there is exactly the same package two lines below

Reply:
In my local it fixed , I just didn't commit it.